### PR TITLE
CONN-1462: Added code to register event loggers

### PR DESCRIPTION
**Pull request by nareshcgi**
Added code to register event loggers to fix the JBoss event logging issue.

### CODE REVIEW

File: Product/Production/Services/DirectCore/src/main/java/gov/hhs/fha/nhinc/direct/DirectAdapterFactory.java [hunk 1]
```diff
@@ -47,6 +48,8 @@ public void registerHandlers() {
         //initialize the HibernateUtil when the Direct Servlet is initialized.. DO NOT Remove this.
         SessionFactory session = gov.hhs.fha.nhinc.event.persistence.HibernateUtil.getSessionFactory();
         session = gov.hhs.fha.nhinc.direct.messagemonitoring.persistence.HibernateUtil.getSessionFactory();
+        LOG.debug("Registering event Loggers");
+        EventLoggerFactory.getInstance().registerLoggers();
         LOG.debug("Registering handlers...");
```
Comment (jsmith2) [inline Product/Production/Services/DirectCore/src/main/java/gov/hhs/fha/nhinc/direct/DirectAdapterFactory.java]: These debug statements should either be changed to trace or removed.

**Comment by jsmith2** [comment]
Reviewed and good with merge.

**Comment by alameluchidambaram** [comment]
I am good with the merge.

**Comment by sailajaa** [comment]
Reviewed and good to merge PR.

**Pull request merged**